packages/next/next-server/server/config.ts

```diff
@@ -47,6 +47,10 @@ const defaultConfig: { [key: string]: any } = {
     profiling: false,
     publicDirectory: false,
     sprFlushToDisk: true,
+    deferPageScript: false,
```

Comment:
I feel like this should be 1 option instead of 4. The goal is to remove this option as soon as it's verified it improves metrics.

Reply:
I could make it one option.

I made it separate flags originally because I wanted to allow as much flexibility as possible. There might be some scripts that you want to run as soon as the page loads, e.g. an analytics payload.

Reply:
Based on the options you added you wouldn't be able to change that granularly though. It's also hard to single out that specific script or it's added through custom _document already (because they require a custom `<script>` to be inlined)